doc/index.rst

```diff
@@ -3,29 +3,48 @@ PyInstaller Manual
 ==================
 
 :Version: |PyInstallerVersion|
-:Homepage: |Homepage|
+:Homepage: `<https://pyinstaller.org/>`_
 :Contact: pyinstaller@googlegroups.com
 :Authors: David Cortesi, based on structure by Giovanni Bajo & William Caban, based on Gordon McMillan's manual
 :Copyright: This document has been placed in the public domain.
 
 
-|PyInstaller| bundles a Python application and all its dependencies into
-a single package.
+PyInstaller bundles a Python application and all its dependencies into a single package.
 The user can run the packaged app without installing a Python interpreter or any modules.
-|PyInstaller| supports Python 3.7 or newer,
-and correctly bundles the major Python packages
+PyInstaller supports Python 3.7 and newer, and correctly bundles many major Python packages
 such as numpy, PyQt, Django, wxPython, and others.
 
-|PyInstaller| is tested against Windows, Mac OS X, and GNU/Linux.
-However, it is not a cross-compiler:
-to make a Windows app you run |PyInstaller| in Windows;
-to make a GNU/Linux app you run it in GNU/Linux, etc.
-|PyInstaller| has been used successfully with
-AIX, Solaris, FreeBSD and OpenBSD
-but testing against them is not part of our continuous integration tests.
+PyInstaller is tested against Windows, MacOS X, and Linux.
+However, it is not a cross-compiler; to make a Windows app you run PyInstaller on Windows,
+and to make a Linux app you run it on Linux, etc.
+
+PyInstaller has been used successfully with AIX, Solaris, FreeBSD and OpenBSD but testing
+against them is not part of our continuous integration tests, and the development team offers
+no guarantee (all code for these platforms comes from external contributions)
+that PyInstaller will work on these platforms or that they will continue to be supported.
+
+
+Quickstart
+__________
+
+Install PyInstaller from PyPI:
+
+.. code-block:: bash
+
+    pip install -U pyinstaller
+
```

Comment:
Can we squeeze in some mention of needing `objcopy` and `objdump` (typically provided by a `binutils` system package) on Linux + BSD.

Reply:
I'll add a link to the requirements page. I'm not actually done with this docs work yet, just trying to get the website + docs merged for the v5 release.